pkgs/applications/networking/bee/bee.nix

```diff
@@ -0,0 +1,79 @@
+{ version ? "release", stdenv, lib, fetchFromGitHub, buildGoModule, coreutils }:
```

Comment:
Why support multiple versions?

Reply:
i'm planning to use Nix to eventually set up bee tests on clusters.

it's also in a flux and running `unstable` sometimes is desirable.